packages/theme-patternfly-org/templates/mdx.js

```diff
@@ -53,7 +53,8 @@ const MDXChildTemplate = ({
       )}
       {beta && (
         <InlineAlert title="Beta feature">
-          This Beta component is currently under review, so please join in and give us your feedback on the <a href="https://forum.patternfly.org/">PatternFly forum</a>.
+          This Beta component is currently under review. It is available for use in product with the caveat that user feedback could result in future breaking changes.
```

Comment:
While what you say in the second sentence is true, I worry that reminding people of this fact will make it less likely for them to use the beta component. In the past, I had gotten feedback that developers were reluctant to user a beta component for fear of breaking changes, but then we don't get feedback.

Reply:
Agree with @mcarrano here - we want to encourage adoption with an understanding that they may continue to be improved upon.

Reply:
hm... i just want to make sure we are also being honest. 
since what I stated is actually my understanding of the definition of a beta component.
Would it be better for the first two sentences to read, `This Beta component is currently under review and is still open for further evolution. It is available for use in product.`

Reply:
I like that better.

Reply:
That sounds great 👍